### PR TITLE
Generate AAR rather than tarball

### PR DESCRIPTION
**Pull request by bassosimone**
Makes it easy to deploy on f-droid.

### CODE REVIEW

**Comment by AntonioLangiu** [comment]
I reviewed this PR and it looks good to me! I made two tests to verify that the new build system works:

 - I tried to generate the AAR with `make dist NDK_BUILD=...` and it worked ✅
 - I tried to create an app and add measurement-kit using gradle and it worked ✅

We are ready to merge!

File: .idea/gradle.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bassosimone) [inline .idea/gradle.xml]: For clarity: file generated by Android studio

File: .idea/runConfigurations.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bassosimone) [inline .idea/runConfigurations.xml]: For clarity: file generated by Android studio

File: gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -0,0 +1,6 @@
+#Mon Apr 03 16:14:04 CEST 2017
```
Comment (bassosimone) [inline gradle/wrapper/gradle-wrapper.properties]: For clarity: file generated by Android studio

File: gradlew [hunk 4]
```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
```
Comment (bassosimone) [inline gradlew]: For clarity: file generated by Android studio

File: gradlew.bat [hunk 5]
```diff
@@ -0,0 +1,90 @@
+@if "%DEBUG%" == "" @echo off
```
Comment (bassosimone) [inline gradlew.bat]: For clarity: file generated by Android studio

File: libmeasurementkit.iml [hunk 6]
```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (bassosimone) [inline libmeasurementkit.iml]: For clarity: file generated by Android studio

File: proguard-rules.pro [hunk 7]
```diff
@@ -0,0 +1,25 @@
+# Add project specific ProGuard rules here.
```
Comment (bassosimone) [inline proguard-rules.pro]: For clarity: file generated by Android studio

**Pull request merged**